Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+It should look something like this for ABP:
+
+```json
+{
+  "deviceId": "BE7A00000000888F",
+  "etag": "AAAAAAAAAAs=",
+  "deviceEtag": "NzMzMTE3MTAz",
+  "status": "enabled",
+  "statusUpdateTime": "0001-01-01T00:00:00",
+  "connectionState": "Disconnected",
+  "lastActivityTime": "2018-08-06T15:16:32.0658492",
+  "cloudToDeviceMessageCount": 0,
+  "authenticationType": "sas",
+  "x509Thumbprint": {
+    "primaryThumbprint": null,
+    "secondaryThumbprint": null
+  },
+  "version": 324,
+  "tags": {
+  
+  },
+  "properties": {
+    "desired": {
+      "AppSKey": "2B7E151628AED2A6ABF7158809CF4F3C",
+      "NwkSKey": "1B6E151628AED2A6ABF7158809CF4F2C",
+      "DevAddr": "0028B9B9",
+      "SensorDecoder": "",
+      "GatewayID": "",
+      "$metadata": {
+        "$lastUpdated": "2018-03-28T06:12:46.1007943Z"
+      },
+      "$version": 1
+    },
+    "reported": {
+      "$metadata": {
+        "$lastUpdated": "2018-08-06T15:16:32.2689851Z"
+      },
+      "$version": 313
+    }
+  }
+}
+```
+
+- Click Save
+- Turn on the device and you are ready to go
+
+### Optional device properties
+
+Customization's to lora devices are set by creating specific twin desired properties on the device. The following customization's are available:
+
+|Name|Description|Configuration|When to use|
+|-|-|-|-|
+|Enable/disable downstream messages|Allows disabling the downstream (cloud to device) for a device. By default downstream messages are enabled| Add twin desired property `"Downlink": false` to disable downstream messages. The absence of the twin property or setting value to `true` will enable downlink messages.|Disabling downlink on devices decreases message processing latency, since the network server will not look for cloud to device messages when an uplink is received. Only disable it in devices that are not expecting messages from cloud. Acknowledgement of confirmed upstream are sent to devices even when downlink is set to false|
+|Preferred receive window|Allows setting the device preferred receive window (RX1 or RX2). The default preferred receive window is 1| Add twin desired property `"PreferredWindow": 2` sets RX2 as preferred window. The absence of the twin property or setting the value to `1` will set RX1 as preferred window.|Using the second receive window increases the chances that the end application can process the upstream message and send a cloud to device message to the lora device without requiring and additional upstream message. Basically completing the round trip in less than 2 seconds.|
+
+**Important**: changes made to twin desired properties in devices that are already connected will only take effect once the network server is restarted or [cache is cleared](#cache-clearing).
+
+### Decoders
+
+The SensorDecoder tag is used to define which method will be used to decode the LoRa payload. If you leave it out or empty it will send the raw decrypted payload in the data field of the json message as Base64 encoded value to IoT Hub.
```

Comment:
add comma "," in the second sentance after the word "empty"... : 

If you leave it out or empty, it will send the raw decrypted payload in the data field of the json message as Base64 encoded value to IoT Hub.